### ISSUE TITLE
Question: Is Fantasy-Land compliant, and to what degree?

### ISSUE THREAD

**Issue by Deluxo** [MeetMartin/lambda#1]
Just the title.

P.S. Would be nice to see coverage badge.

**Comment by MeetMartin** [comment]
Hi Deluxo.

I am familiar with fantasy land but I have never thought about any kind of compliance. The monads within the framework fully follow the rules for functors, monads, and applicatives and provide the necessary functions. I have been using it in other projects first to make sure that all issues are worked out.

If fantasy land compliance is something that is important to you can you please tell me more about why?

If you mean test coverage there is 100 % test coverage by Jest and the tests are available within the repository to review. I need to figure out how to get a badge first.

Thanks
Martin

**Comment by Deluxo** [comment]
This weekend is the first time I spend my proper time learning about FP in general.
In fact, Your tweet was the hook I got on.

For what I've read, and my humble research about other libraries, the question itself isn't that worthwhile.

Your library is really sweet compared to what already was available.

Just after I wrote the question, I've run the tests myself...

My next step will be practical exercises refactoring my code-base in FP style with Your library, as I've already tried ramda, folktale, monet and few others.

Seems like that I'm also will stick to it.
